site/content/en/docs/manual/advanced/formats/format-cityscapes.md

```diff
@@ -0,0 +1,142 @@
+---
+linkTitle: 'Cityscapes'
+weight: 16
+---
+
+# [Cityscapes](https://www.cityscapes-dataset.com/login/)
+
+- [Format specification](https://github.com/mcordts/cityscapesScripts#the-cityscapes-dataset)
+
+- Supported annotations
+
+  - Polygons (segmentation task)
+
+- Supported attributes
+  - 'is_crowd' (boolean, should be defined for labels as `checkbox` -es)
+    Specifies if the annotation label can distinguish between different instances.
+    If False, the annotation id field encodes the instance id.
+
+# Cityscapes export
+
+Downloaded file: a zip archive of the following structure:
+
+```
+.
+├── label_color.txt
+├── gtFine
+│   ├── <subset_name>
+│   │   └── <city_name>
+│   │       ├── image_0_gtFine_instanceIds.png
+│   │       ├── image_0_gtFine_color.png
+│   │       ├── image_0_gtFine_labelIds.png
+│   │       ├── image_1_gtFine_instanceIds.png
+│   │       ├── image_1_gtFine_color.png
+│   │       ├── image_1_gtFine_labelIds.png
+│   │       ├── ...
+└── imgsFine  # if saving images was requested
+    └── leftImg8bit
+        ├── <subset_name>
+        │   └── <city_name>
+        │       ├── image_0_leftImg8bit.png
```

Comment:
@kirill-sizov , @zhiltsov-max , Just a question. Should we really add _leftImg8bit suffix to an image name?

Reply:
I think, yes, we should, because [original dataset](https://www.cityscapes-dataset.com/downloads/) and [scripts](https://github.com/mcordts/cityscapesScripts) provide such naming of images.